packages/react-hooks/src/hooks/use-cart/actions/initCart.ts

```diff
@@ -0,0 +1,25 @@
+import { CartItem } from '../../common/types';
+
+import { InitCartAction, CartState } from '~/hooks/use-cart/use-cart.types';
+
+const initCart: InitCartFunction = (
+  state: CartState,
+  action: InitCartAction
+) => {
+  const cart: CartItem[] = [...action.payload];
+
```

Comment:
We could use `setCacheItem` in this action to stay persist the new sate of the cart in browser storage. That will keep this action consistent with other actions which modify the cart (`actions/addToCart.ts` is a good example).